ash/src/vk/features.rs

```diff
@@ -19,31 +19,22 @@ impl ::std::clone::Clone for StaticFn {
     }
 }
 impl StaticFn {
-    pub fn load<F>(mut _f: F) -> Self
+    pub fn load<F>(mut _f: F) -> std::result::Result<Self, MissingEntryPoint>
     where
         F: FnMut(&::std::ffi::CStr) -> *const c_void,
     {
-        StaticFn {
+        Ok(StaticFn {
             get_instance_proc_addr: unsafe {
-                extern "system" fn get_instance_proc_addr(
-                    _instance: Instance,
-                    _p_name: *const c_char,
-                ) -> PFN_vkVoidFunction {
-                    panic!(concat!(
-                        "Unable to load ",
-                        stringify!(get_instance_proc_addr)
-                    ))
-                }
                 let cname =
                     ::std::ffi::CStr::from_bytes_with_nul_unchecked(b"vkGetInstanceProcAddr\0");
                 let val = _f(cname);
                 if val.is_null() {
-                    get_instance_proc_addr
+                    return Err(MissingEntryPoint(cname.to_owned()));
```

Comment:
You could possibly store `&'static str` here of the direct string literal `"vkGetInstanceProcAddr"` instead of copying, though for an unlikely error clause that might not matter too much.

Reply:
Good point, it's very unlikely that you ever want to dynamically generate the function name, so a `&'static` will do.